channel/consensus_channel/consensus_channel.go

```diff
@@ -262,6 +262,10 @@ type Balance struct {
 	amount      *big.Int
 }
 
+func (b Balance) Equal(b2 Balance) bool {
+	return bytes.Equal(b.destination.Bytes(), b2.destination.Bytes()) && b.amount.Cmp(b2.amount) == 0
```

Comment:
You can use our own `types.Equal` function, which is marginally nicer to read than `.Cmp()==0`.

Reply:
Updated in https://github.com/statechannels/go-nitro/pull/618/commits/9550fc16fa1cafadbcdee1f76cc65cf4e32c7383